### DISCUSSION TITLE
Has the MariaDB issue been resolved?

### DISCUSSION THREAD

**Discussion by Xykrotik** [benjaminjonard/koillection#854] [Q&A] [answered]
I was searching through the 'Issues' tab for anything relating to compatibility with MariaDB. According to #429 & #253 there were issues (on MariaDB's end) getting things setup and to work properly. Is anyone aware if the incompatibility issue was ever resolved on MariaDB's end?

**Comment by benjaminjonard** [comment, marked as answer]
Last time I checked the issue wasn't solved on MariaDB's end. But I made changes to the code and MariaDB should work just fine (since Koillection release 1.3.15)
